Project.toml

```
@@ -27,7 +27,7 @@ MultipleTesting = "0.5, 0.6"
 NaturalSort = "1.0"
 OrderedCollections = "1"
 Printf = "1"
-SHA = "0.7"
+SHA = "0.7, 1.7"
```

Based on https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958#update-november-9th-2023-2 (and in particular the update from 9 November) I think the safest approach would be to allow
```suggestion
SHA = "<0.0.1, 0.7, 1"
```